webui-ros-joystick/launch/webui.launch

```diff
@@ -2,19 +2,21 @@
 
     <arg name="node_name" default="node_server" />
     <arg name="wait_nodes" default="" />
+    <arg name="cmd_vel_topic" default="/cmd_vel" />
     <arg name="max_lin_vel" default="1.0" />
     <arg name="max_ang_vel" default="1.0" />
     <arg name="max_lin_accel" default="2.0" />
     <arg name="max_ang_accel" default="2.0" />
     <arg name="e_stop" default="false" />
 
     <node pkg="webui-ros-joystick" type="node_server.sh" name="$(arg node_name)" output="screen">
+        <param name="wait_nodes" value="$(arg wait_nodes)" />
+        <param name="cmd_vel_topic" value="$(arg cmd_vel_topic)" />
```

Comment:
There is remapping in ROS for a reason. Passing topic name as a param is not the best practice

Reply:
Generally yes, but using param for this topic is more versatile e.g. useful when using yaml config files. Adding params for crucial topics is used for example in Nav2.